src/main/java/seedu/address/logic/commands/ServeCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_NO_SUCH_BORROWERID;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.borrower.Borrower;
+import seedu.address.model.borrower.BorrowerId;
+
+/**
+ * Opens a serving session for a borrower and allows the borrower to start borrower book
+ */
+public class ServeCommand extends Command {
```

Comment:
Have you added ServeCommandParser and updated the CatalogParser?

Reply:
Not yet! Will include in next PR. This PR is to assist holyin in his implementations.